files/jira.rb

```diff
@@ -111,6 +115,10 @@ def project
     @event['check']['project'] || team_data('project')
   end
 
+  def priority
+    @event['check']['priority'] || team_data('default_priority')
```

Comment:
Where does this default_priority come from, and how can we be sure it's set? (I.e. do we need a last ditch fallback default?)

Reply:
The `default_priority` is just something that could be defined per-team, like in `sensu_handlers::teams`, since some teams have things like `ad_backend`/`ad_backend_business_hours_critical`/`ad_backend_noncritical` or `ad_data_insights_p0`/`ad_data_insights_p1`, which should be different priorities. These could have the priorities added in directly per team so that the created tickets are made as the correct priority by default if not overridden by a specific check's priority.

As for if this needs a fallback value, I don't believe it should. If both are `nil`, then that's fine and the priority just won't get set by jira, since `handler_settings['priority_map'].include?(priority)` will be false. Very good question though, I didn't make this very clear from the code itself.

Reply:
Cool, got it. Can you drop a line or two of comments in to explain this for the next person who comes along, then shipit!

Reply:
It's probably just too early, but I can't find the approve button :) Feel free to add comment and ship...